client/core/core.go

```diff
@@ -4170,21 +4192,26 @@ func (c *Core) Login(pw []byte) (*LoginResult, error) {
 	if walletCount > 0 {
 		c.log.Infof("Connected to %d of %d wallets.", connectCount, walletCount)
 	}
+}
 
-	// Load trades from the DB, unlock required wallets for active orders, and
-	// populate the dc.trades map. NOTE: repeated login may involve redundant DB
-	// load of trades already in the dc.trades map, but new required wallets
-	// may be accessible this time.
-	c.resolveActiveTrades(crypter)
-	// Unlock account keys and attempt to authorize with each dexConnection.
-	// Even if connect fails, this prepares the account private keys so authDEX
-	// triggered by the reconnect handler can succeed.
-
-	if err := c.loadBotPrograms(); err != nil {
-		c.log.Errorf("Error loading bot programs: %v", err)
+// AuthDEXConnections connects to the each of the DEX servers in the conns map
+// and authenitcates the connection.
+func (c *Core) AuthDEXConnections(pw []byte) ([]*DEXBrief, error) {
+	crypter, err := c.encryptionKey(pw)
+	if err != nil {
+		return nil, err
 	}
 
-	return loginResult(), nil
+	return c.initializeDEXConnections(crypter), nil
+}
+
+// GetNotifications loads the latest notifications from the db.
+func (c *Core) GetNotifications(n int) ([]*db.Notification, error) {
```

Comment:
`s/GetNotifications/Notifications/g`

A "get" prefix on method names is rarely valuable, unless is has really needed to convey there is some remote fetching going on, but usually not even in that case.

It's idiomatic to elide the get [for "getters"](https://go.dev/doc/effective_go#Getters), and the reasoning applies to most other kinds of methods too.